README.md

```diff
@@ -1 +1,31 @@
-# QuantumNLDiffEq.jl
\ No newline at end of file
+# QuantumNLDiffEq.jl
+
+### Installation
+
+```julia
+]add add YaoBlocks#master https://github.com/VarLad/QuantumNLDiffEq.jl
+```
+
+### Usage
+
+```julia
+
+julia> begin
+           using Yao, QuantumNLDiffEq
+           M = range(start=0; stop=0.9, length=20)
+           cost = sum([put(6, i=>Z) for i in 1:6])
+           var = dispatch(EasyBuild.variational_circuit(6,5), :random)
+           quantum_feature_map_circuit = chain(6, [put(i=>Ry(0)) for i in 1:6])
+           params = parameters(var)
+           u_0 = 1.0
+           N = 6
+       end
+       
+julia> train!(quantum_feature_map_circuit, cost, var, M, N, 1.0, params)
+
+julia> e = [expect(cost, zero_state(N)=>QuantumNLDiffEq.new_circuit(quantum_feature_map_circuit, var, i, params, N; mapping=QuantumNLDiffEq.Chebyshev())) .+ u_0 .- expect(cost, zero_state(N)=>QuantumNLDiffEq.new_circuit(quantum_feature_map_circuit, var, M[1], params, N; mapping=QuantumNLDiffEq.Chebyshev()))  for i in M]
+
+julia> import Plots
+
+julia> Plots.plot(M, real.(e))
+```
```

Comment:
paste a plot into the PR as a comment, then use its link to add the plot here.

Reply:
<?xml version="1.0" encoding="utf-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="600" height="400" viewBox="0 0 2400 1600">
<defs>
  <clipPath id="clip600">
    <rect x="0" y="0" width="2400" height="1600"/>
  </clipPath>
</defs>
<path clip-path="url(#clip600)" d="
M0 1600 L2400 1600 L2400 0 L0 0  Z
  " fill="#ffffff" fill-rule="evenodd" fill-opacity="1"/>
<defs>
  <clipPath id="clip601">
    <rect x="480" y="0" width="1681" height="1600"/>
  </clipPath>
</defs>
<path clip-path="url(#clip600)" d="
M192.941 1486.45 L2352.76 1486.45 L2352.76 47.2441 L192.941 47.2441  Z
  " fill="#ffffff" fill-rule="evenodd" fill-opacity="1"/>
<defs>
  <clipPath id="clip602">
    <rect x="192" y="47" width="2161" height="1440"/>
  </clipPath>
</defs>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  254.067,1486.45 254.067,47.2441 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  706.859,1486.45 706.859,47.2441 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  1159.65,1486.45 1159.65,47.2441 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  1612.44,1486.45 1612.44,47.2441 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  2065.23,1486.45 2065.23,47.2441 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,1486.45 2352.76,1486.45 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  254.067,1486.45 254.067,1467.55 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  706.859,1486.45 706.859,1467.55 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  1159.65,1486.45 1159.65,1467.55 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  1612.44,1486.45 1612.44,1467.55 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  2065.23,1486.45 2065.23,1467.55 
  "/>
<path clip-path="url(#clip600)" d="M231.452 1517.37 Q227.841 1517.37 226.012 1520.93 Q224.207 1524.47 224.207 1531.6 Q224.207 1538.71 226.012 1542.27 Q227.841 1545.82 231.452 1545.82 Q235.086 1545.82 236.892 1542.27 Q238.72 1538.71 238.72 1531.6 Q238.72 1524.47 236.892 1520.93 Q235.086 1517.37 231.452 1517.37 M231.452 1513.66 Q237.262 1513.66 240.318 1518.27 Q243.396 1522.85 243.396 1531.6 Q243.396 1540.33 240.318 1544.94 Q237.262 1549.52 231.452 1549.52 Q225.642 1549.52 222.563 1544.94 Q219.507 1540.33 219.507 1531.6 Q219.507 1522.85 222.563 1518.27 Q225.642 1513.66 231.452 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M251.614 1542.97 L256.498 1542.97 L256.498 1548.85 L251.614 1548.85 L251.614 1542.97 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M276.683 1517.37 Q273.072 1517.37 271.243 1520.93 Q269.438 1524.47 269.438 1531.6 Q269.438 1538.71 271.243 1542.27 Q273.072 1545.82 276.683 1545.82 Q280.317 1545.82 282.123 1542.27 Q283.952 1538.71 283.952 1531.6 Q283.952 1524.47 282.123 1520.93 Q280.317 1517.37 276.683 1517.37 M276.683 1513.66 Q282.493 1513.66 285.549 1518.27 Q288.627 1522.85 288.627 1531.6 Q288.627 1540.33 285.549 1544.94 Q282.493 1549.52 276.683 1549.52 Q270.873 1549.52 267.794 1544.94 Q264.739 1540.33 264.739 1531.6 Q264.739 1522.85 267.794 1518.27 Q270.873 1513.66 276.683 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M685.042 1517.37 Q681.431 1517.37 679.602 1520.93 Q677.797 1524.47 677.797 1531.6 Q677.797 1538.71 679.602 1542.27 Q681.431 1545.82 685.042 1545.82 Q688.676 1545.82 690.482 1542.27 Q692.31 1538.71 692.31 1531.6 Q692.31 1524.47 690.482 1520.93 Q688.676 1517.37 685.042 1517.37 M685.042 1513.66 Q690.852 1513.66 693.908 1518.27 Q696.986 1522.85 696.986 1531.6 Q696.986 1540.33 693.908 1544.94 Q690.852 1549.52 685.042 1549.52 Q679.232 1549.52 676.153 1544.94 Q673.098 1540.33 673.098 1531.6 Q673.098 1522.85 676.153 1518.27 Q679.232 1513.66 685.042 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M705.204 1542.97 L710.088 1542.97 L710.088 1548.85 L705.204 1548.85 L705.204 1542.97 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M724.301 1544.91 L740.62 1544.91 L740.62 1548.85 L718.676 1548.85 L718.676 1544.91 Q721.338 1542.16 725.921 1537.53 Q730.528 1532.88 731.708 1531.53 Q733.954 1529.01 734.833 1527.27 Q735.736 1525.51 735.736 1523.82 Q735.736 1521.07 733.792 1519.33 Q731.87 1517.6 728.769 1517.6 Q726.569 1517.6 724.116 1518.36 Q721.685 1519.13 718.907 1520.68 L718.907 1515.95 Q721.732 1514.82 724.185 1514.24 Q726.639 1513.66 728.676 1513.66 Q734.046 1513.66 737.241 1516.35 Q740.435 1519.03 740.435 1523.52 Q740.435 1525.65 739.625 1527.57 Q738.838 1529.47 736.731 1532.07 Q736.153 1532.74 733.051 1535.95 Q729.949 1539.15 724.301 1544.91 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1136.79 1517.37 Q1133.18 1517.37 1131.35 1520.93 Q1129.55 1524.47 1129.55 1531.6 Q1129.55 1538.71 1131.35 1542.27 Q1133.18 1545.82 1136.79 1545.82 Q1140.43 1545.82 1142.23 1542.27 Q1144.06 1538.71 1144.06 1531.6 Q1144.06 1524.47 1142.23 1520.93 Q1140.43 1517.37 1136.79 1517.37 M1136.79 1513.66 Q1142.6 1513.66 1145.66 1518.27 Q1148.74 1522.85 1148.74 1531.6 Q1148.74 1540.33 1145.66 1544.94 Q1142.6 1549.52 1136.79 1549.52 Q1130.98 1549.52 1127.9 1544.94 Q1124.85 1540.33 1124.85 1531.6 Q1124.85 1522.85 1127.9 1518.27 Q1130.98 1513.66 1136.79 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1156.95 1542.97 L1161.84 1542.97 L1161.84 1548.85 L1156.95 1548.85 L1156.95 1542.97 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1184.87 1518.36 L1173.06 1536.81 L1184.87 1536.81 L1184.87 1518.36 M1183.64 1514.29 L1189.52 1514.29 L1189.52 1536.81 L1194.45 1536.81 L1194.45 1540.7 L1189.52 1540.7 L1189.52 1548.85 L1184.87 1548.85 L1184.87 1540.7 L1169.27 1540.7 L1169.27 1536.19 L1183.64 1514.29 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1589.75 1517.37 Q1586.13 1517.37 1584.31 1520.93 Q1582.5 1524.47 1582.5 1531.6 Q1582.5 1538.71 1584.31 1542.27 Q1586.13 1545.82 1589.75 1545.82 Q1593.38 1545.82 1595.19 1542.27 Q1597.01 1538.71 1597.01 1531.6 Q1597.01 1524.47 1595.19 1520.93 Q1593.38 1517.37 1589.75 1517.37 M1589.75 1513.66 Q1595.56 1513.66 1598.61 1518.27 Q1601.69 1522.85 1601.69 1531.6 Q1601.69 1540.33 1598.61 1544.94 Q1595.56 1549.52 1589.75 1549.52 Q1583.94 1549.52 1580.86 1544.94 Q1577.8 1540.33 1577.8 1531.6 Q1577.8 1522.85 1580.86 1518.27 Q1583.94 1513.66 1589.75 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1609.91 1542.97 L1614.79 1542.97 L1614.79 1548.85 L1609.91 1548.85 L1609.91 1542.97 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M1635.56 1529.7 Q1632.41 1529.7 1630.56 1531.86 Q1628.73 1534.01 1628.73 1537.76 Q1628.73 1541.49 1630.56 1543.66 Q1632.41 1545.82 1635.56 1545.82 Q1638.7 1545.82 1640.53 1543.66 Q1642.38 1541.49 1642.38 1537.76 Q1642.38 1534.01 1640.53 1531.86 Q1638.7 1529.7 1635.56 1529.7 M1644.84 1515.05 L1644.84 1519.31 Q1643.08 1518.48 1641.27 1518.04 Q1639.49 1517.6 1637.73 1517.6 Q1633.1 1517.6 1630.65 1520.72 Q1628.22 1523.85 1627.87 1530.17 Q1629.24 1528.15 1631.3 1527.09 Q1633.36 1526 1635.83 1526 Q1641.04 1526 1644.05 1529.17 Q1647.08 1532.32 1647.08 1537.76 Q1647.08 1543.08 1643.93 1546.3 Q1640.79 1549.52 1635.56 1549.52 Q1629.56 1549.52 1626.39 1544.94 Q1623.22 1540.33 1623.22 1531.6 Q1623.22 1523.41 1627.11 1518.55 Q1631 1513.66 1637.55 1513.66 Q1639.31 1513.66 1641.09 1514.01 Q1642.89 1514.36 1644.84 1515.05 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M2042.66 1517.37 Q2039.05 1517.37 2037.22 1520.93 Q2035.42 1524.47 2035.42 1531.6 Q2035.42 1538.71 2037.22 1542.27 Q2039.05 1545.82 2042.66 1545.82 Q2046.3 1545.82 2048.1 1542.27 Q2049.93 1538.71 2049.93 1531.6 Q2049.93 1524.47 2048.1 1520.93 Q2046.3 1517.37 2042.66 1517.37 M2042.66 1513.66 Q2048.47 1513.66 2051.53 1518.27 Q2054.61 1522.85 2054.61 1531.6 Q2054.61 1540.33 2051.53 1544.94 Q2048.47 1549.52 2042.66 1549.52 Q2036.85 1549.52 2033.78 1544.94 Q2030.72 1540.33 2030.72 1531.6 Q2030.72 1522.85 2033.78 1518.27 Q2036.85 1513.66 2042.66 1513.66 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M2062.83 1542.97 L2067.71 1542.97 L2067.71 1548.85 L2062.83 1548.85 L2062.83 1542.97 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M2087.9 1532.44 Q2084.56 1532.44 2082.64 1534.22 Q2080.74 1536 2080.74 1539.13 Q2080.74 1542.25 2082.64 1544.03 Q2084.56 1545.82 2087.9 1545.82 Q2091.23 1545.82 2093.15 1544.03 Q2095.07 1542.23 2095.07 1539.13 Q2095.07 1536 2093.15 1534.22 Q2091.25 1532.44 2087.9 1532.44 M2083.22 1530.45 Q2080.21 1529.7 2078.52 1527.64 Q2076.85 1525.58 2076.85 1522.62 Q2076.85 1518.48 2079.79 1516.07 Q2082.76 1513.66 2087.9 1513.66 Q2093.06 1513.66 2096 1516.07 Q2098.94 1518.48 2098.94 1522.62 Q2098.94 1525.58 2097.25 1527.64 Q2095.58 1529.7 2092.59 1530.45 Q2095.97 1531.23 2097.85 1533.52 Q2099.75 1535.82 2099.75 1539.13 Q2099.75 1544.15 2096.67 1546.83 Q2093.61 1549.52 2087.9 1549.52 Q2082.18 1549.52 2079.1 1546.83 Q2076.04 1544.15 2076.04 1539.13 Q2076.04 1535.82 2077.94 1533.52 Q2079.84 1531.23 2083.22 1530.45 M2081.51 1523.06 Q2081.51 1525.75 2083.17 1527.25 Q2084.86 1528.76 2087.9 1528.76 Q2090.9 1528.76 2092.59 1527.25 Q2094.31 1525.75 2094.31 1523.06 Q2094.31 1520.38 2092.59 1518.87 Q2090.9 1517.37 2087.9 1517.37 Q2084.86 1517.37 2083.17 1518.87 Q2081.51 1520.38 2081.51 1523.06 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  192.941,1290.36 2352.76,1290.36 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  192.941,889.563 2352.76,889.563 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  192.941,488.77 2352.76,488.77 
  "/>
<polyline clip-path="url(#clip602)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:2; stroke-opacity:0.1; fill:none" points="
  192.941,87.9763 2352.76,87.9763 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,1486.45 192.941,47.2441 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,1290.36 211.838,1290.36 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,889.563 211.838,889.563 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,488.77 211.838,488.77 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  192.941,87.9763 211.838,87.9763 
  "/>
<path clip-path="url(#clip600)" d="M50.9921 1290.81 L80.6679 1290.81 L80.6679 1294.74 L50.9921 1294.74 L50.9921 1290.81 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M100.76 1276.16 Q97.1493 1276.16 95.3206 1279.72 Q93.515 1283.26 93.515 1290.39 Q93.515 1297.5 95.3206 1301.06 Q97.1493 1304.6 100.76 1304.6 Q104.395 1304.6 106.2 1301.06 Q108.029 1297.5 108.029 1290.39 Q108.029 1283.26 106.2 1279.72 Q104.395 1276.16 100.76 1276.16 M100.76 1272.45 Q106.571 1272.45 109.626 1277.06 Q112.705 1281.64 112.705 1290.39 Q112.705 1299.12 109.626 1303.72 Q106.571 1308.31 100.76 1308.31 Q94.9502 1308.31 91.8715 1303.72 Q88.816 1299.12 88.816 1290.39 Q88.816 1281.64 91.8715 1277.06 Q94.9502 1272.45 100.76 1272.45 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M120.922 1301.76 L125.807 1301.76 L125.807 1307.64 L120.922 1307.64 L120.922 1301.76 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M136.038 1273.08 L154.394 1273.08 L154.394 1277.01 L140.32 1277.01 L140.32 1285.48 Q141.339 1285.14 142.357 1284.97 Q143.376 1284.79 144.394 1284.79 Q150.181 1284.79 153.561 1287.96 Q156.941 1291.13 156.941 1296.55 Q156.941 1302.13 153.468 1305.23 Q149.996 1308.31 143.677 1308.31 Q141.501 1308.31 139.232 1307.94 Q136.987 1307.57 134.58 1306.83 L134.58 1302.13 Q136.663 1303.26 138.885 1303.82 Q141.107 1304.37 143.584 1304.37 Q147.589 1304.37 149.927 1302.27 Q152.265 1300.16 152.265 1296.55 Q152.265 1292.94 149.927 1290.83 Q147.589 1288.72 143.584 1288.72 Q141.709 1288.72 139.834 1289.14 Q137.982 1289.56 136.038 1290.44 L136.038 1273.08 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M99.765 875.362 Q96.1539 875.362 94.3252 878.927 Q92.5197 882.468 92.5197 889.598 Q92.5197 896.704 94.3252 900.269 Q96.1539 903.811 99.765 903.811 Q103.399 903.811 105.205 900.269 Q107.033 896.704 107.033 889.598 Q107.033 882.468 105.205 878.927 Q103.399 875.362 99.765 875.362 M99.765 871.658 Q105.575 871.658 108.631 876.265 Q111.709 880.848 111.709 889.598 Q111.709 898.325 108.631 902.931 Q105.575 907.515 99.765 907.515 Q93.9549 907.515 90.8762 902.931 Q87.8206 898.325 87.8206 889.598 Q87.8206 880.848 90.8762 876.265 Q93.9549 871.658 99.765 871.658 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M119.927 900.964 L124.811 900.964 L124.811 906.843 L119.927 906.843 L119.927 900.964 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M144.996 875.362 Q141.385 875.362 139.556 878.927 Q137.751 882.468 137.751 889.598 Q137.751 896.704 139.556 900.269 Q141.385 903.811 144.996 903.811 Q148.63 903.811 150.436 900.269 Q152.265 896.704 152.265 889.598 Q152.265 882.468 150.436 878.927 Q148.63 875.362 144.996 875.362 M144.996 871.658 Q150.806 871.658 153.862 876.265 Q156.941 880.848 156.941 889.598 Q156.941 898.325 153.862 902.931 Q150.806 907.515 144.996 907.515 Q139.186 907.515 136.107 902.931 Q133.052 898.325 133.052 889.598 Q133.052 880.848 136.107 876.265 Q139.186 871.658 144.996 871.658 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M100.76 474.568 Q97.1493 474.568 95.3206 478.133 Q93.515 481.675 93.515 488.805 Q93.515 495.911 95.3206 499.476 Q97.1493 503.017 100.76 503.017 Q104.395 503.017 106.2 499.476 Q108.029 495.911 108.029 488.805 Q108.029 481.675 106.2 478.133 Q104.395 474.568 100.76 474.568 M100.76 470.865 Q106.571 470.865 109.626 475.471 Q112.705 480.055 112.705 488.805 Q112.705 497.531 109.626 502.138 Q106.571 506.721 100.76 506.721 Q94.9502 506.721 91.8715 502.138 Q88.816 497.531 88.816 488.805 Q88.816 480.055 91.8715 475.471 Q94.9502 470.865 100.76 470.865 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M120.922 500.17 L125.807 500.17 L125.807 506.05 L120.922 506.05 L120.922 500.17 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M136.038 471.49 L154.394 471.49 L154.394 475.425 L140.32 475.425 L140.32 483.897 Q141.339 483.55 142.357 483.388 Q143.376 483.203 144.394 483.203 Q150.181 483.203 153.561 486.374 Q156.941 489.545 156.941 494.962 Q156.941 500.541 153.468 503.642 Q149.996 506.721 143.677 506.721 Q141.501 506.721 139.232 506.351 Q136.987 505.98 134.58 505.24 L134.58 500.541 Q136.663 501.675 138.885 502.23 Q141.107 502.786 143.584 502.786 Q147.589 502.786 149.927 500.679 Q152.265 498.573 152.265 494.962 Q152.265 491.351 149.927 489.244 Q147.589 487.138 143.584 487.138 Q141.709 487.138 139.834 487.555 Q137.982 487.971 136.038 488.851 L136.038 471.49 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M90.5752 101.321 L98.2141 101.321 L98.2141 74.9555 L89.904 76.6222 L89.904 72.3629 L98.1678 70.6963 L102.844 70.6963 L102.844 101.321 L110.483 101.321 L110.483 105.256 L90.5752 105.256 L90.5752 101.321 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M119.927 99.3767 L124.811 99.3767 L124.811 105.256 L119.927 105.256 L119.927 99.3767 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M144.996 73.775 Q141.385 73.775 139.556 77.3398 Q137.751 80.8814 137.751 88.011 Q137.751 95.1174 139.556 98.6822 Q141.385 102.224 144.996 102.224 Q148.63 102.224 150.436 98.6822 Q152.265 95.1174 152.265 88.011 Q152.265 80.8814 150.436 77.3398 Q148.63 73.775 144.996 73.775 M144.996 70.0713 Q150.806 70.0713 153.862 74.6777 Q156.941 79.261 156.941 88.011 Q156.941 96.7378 153.862 101.344 Q150.806 105.928 144.996 105.928 Q139.186 105.928 136.107 101.344 Q133.052 96.7378 133.052 88.011 Q133.052 79.261 136.107 74.6777 Q139.186 70.0713 144.996 70.0713 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><polyline clip-path="url(#clip602)" style="stroke:#009af9; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  254.067,87.9763 361.308,219.416 468.548,402.994 575.788,622.321 683.028,855.465 790.268,1077.41 897.508,1263.14 1004.75,1390.95 1111.99,1445.72 1219.23,1421.48 
  1326.47,1323.02 1433.71,1165.89 1540.95,974.708 1648.19,779.625 1755.43,611.232 1862.67,494.678 1969.91,444.3 2077.15,460.704 2184.39,532.631 2291.63,645.278 
  
  "/>
<path clip-path="url(#clip600)" d="
M1988.35 198.898 L2280.76 198.898 L2280.76 95.2176 L1988.35 95.2176  Z
  " fill="#ffffff" fill-rule="evenodd" fill-opacity="1"/>
<polyline clip-path="url(#clip600)" style="stroke:#000000; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  1988.35,198.898 2280.76,198.898 2280.76,95.2176 1988.35,95.2176 1988.35,198.898 
  "/>
<polyline clip-path="url(#clip600)" style="stroke:#009af9; stroke-linecap:butt; stroke-linejoin:round; stroke-width:4; stroke-opacity:1; fill:none" points="
  2012.35,147.058 2156.34,147.058 
  "/>
<path clip-path="url(#clip600)" d="M2194.18 166.745 Q2192.37 171.375 2190.66 172.787 Q2188.95 174.199 2186.08 174.199 L2182.67 174.199 L2182.67 170.634 L2185.17 170.634 Q2186.93 170.634 2187.91 169.8 Q2188.88 168.967 2190.06 165.865 L2190.82 163.921 L2180.34 138.412 L2184.85 138.412 L2192.95 158.689 L2201.05 138.412 L2205.57 138.412 L2194.18 166.745 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /><path clip-path="url(#clip600)" d="M2212.86 160.402 L2220.5 160.402 L2220.5 134.037 L2212.19 135.703 L2212.19 131.444 L2220.45 129.778 L2225.13 129.778 L2225.13 160.402 L2232.77 160.402 L2232.77 164.338 L2212.86 164.338 L2212.86 160.402 Z" fill="#000000" fill-rule="evenodd" fill-opacity="1" /></svg>

Reply:
![plot1](https://user-images.githubusercontent.com/51269425/160281466-ced2acc9-fb2e-4ad0-a23a-4bc1dfd934a9.svg)

Reply:
@VarLad I cannot reproduce this plot.
![download](https://user-images.githubusercontent.com/18127060/160288840-2487c017-b34f-4743-83f2-82111b557523.png)

I think it would be appropriate to

- [ ] Make test case from README.md example